test/test_um2netcdf.py

```diff
@@ -86,21 +86,23 @@ class DummyCube:
 
     def __init__(self, item_code, var_name=None, attributes=None,
                  units=None, coords=None):
-        self.item_code = item_code
+        self.item_code = item_code  # NB: um2nc adds this at runtime
         self.var_name = var_name or "unknown_var"
-        self.attributes = attributes
-        self.units = units
         self.standard_name = None
         self.long_name = ""
```

Comment:
Does it make sense to treat `standard_name` and `long_name` differently in this case? Why not just set `standard_name` to empty string too?

Reply:
The discrepancy possibly arose from merging ~3 different cube fakes. A very quick check indicated the default "no name" value as `None`, so I've defaulted to that in 53816cb0.